identitykit/src/main/java/com/eldersoss/identitykit/network/volley/VolleyNetworkClient.kt

```diff
@@ -12,7 +12,7 @@ import com.eldersoss.identitykit.network.NetworkResponse
 /**
  * Created by IvanVatov on 8/18/2017.
  */
-class VolleyNetworkClient(val context: Context) : NetworkClient {
+class VolleyNetworkClient(val context: Context, val locale: String?) : NetworkClient {
```

Comment:
This is application logic, that should reside inside the application.
The identity kit can accept something like `additionalHeaders` parameter, which to add to the request.
From there the client app can pass any desired header.